### PR TITLE
Adding snapp divar filimo aparat categories and updating some Iranian domain categories

### PR DESCRIPTION
**Pull request by alirezax5**
snapp.ir may have other domains that I will try to check and if a new domain is found, I will add it

I tried my best that the lists are sorted by name

### CODE REVIEW

**Comment by rootmelo92118** [comment]
I see you add a lot of domain that belongs to snapp, why we don't create a new category `snapp` to save the content of it? And include it in `category-ir`.

**Comment by alirezax5** [comment]
> I see you add a lot of domain that belongs to snapp, why we don't create a new category `snapp` to save the content of it? And include it in `category-ir`.

Hello, I did not do this because the domains related to snapp.ir were used in the category-shopping-ir and category-travel-ir categories.

If it doesn't cause a problem for the project process, I will put it in a new category called snapp

**Comment by alirezax5** [comment]
Added categories:
snapchat
snapp
divar

**Comment by rootmelo92118** [comment]
If `aparat`, `divar`, `filimo` and `snapchat` not belong to Iran. Please separate them into different pull requests.

**Comment by alirezax5** [comment]
> If `aparat`, `divar`, `filimo` and `snapchat` are not belong to Iran. Please separate them into different pull requests.

snapchat was not only related to Iran, I checked and saw that it already existed under the name of snap, so I deleted it

**Review by rootmelo92118 — APPROVED** [review]
@IceCodeNew Do you have any opinion?

**Comment by IceCodeNew** [comment]
> @IceCodeNew I figure this pull request can be merged. Do you have any opinion?

I will review this PR late today.

File: data/snapp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cdnsnapptrip.com
+snapp-box.com
+snapp.cab
+snapp.doctor
+snapp.ir
```
Comment (bootmortis) [inline data/snapp:5]: we could add `snapp.express` here too.
```suggestion
snapp.doctor
snapp.express
snapp.ir
```

File: data/snapp [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+snapp.ninja
+snapp.site
```
Comment (bootmortis) [inline data/snapp:7]: also `snapp.supply` here
```suggestion
snapp.site
snapp.supply
```

**Review by IceCodeNew — APPROVED** [review]

**Comment by IceCodeNew** [comment]
This PR is perfect! Thank everyone for putting great effort into it.
@alirezax5 @rootmelo92118 @bootmortis

**Pull request merged**